2.feature_extraction/cp_analysis_hpc.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+#SBATCH --nodes=1
+#SBATCH --ntasks=6 # number of cores (plates)
+#SBATCH --partition=amilan
+#SBATCH --qos=normal
```

Comment:
This should be changed to `long` as you are running for more than 24 hours. 
* <24 hours amilan = normal
*  >24 hours amilan = long
*  any time on amem = mem
```suggestion
#SBATCH --qos=long
```

Reply:
Thanks for pointing this out! I have updated!